.github/workflows/poetry-test.yml

```diff
@@ -38,6 +38,3 @@ jobs:
 
       - name: 🧪 Run the Import test
         run: poetry run python -c "import supervision; from supervision import assets; from supervision import metrics; print(supervision.__version__)"
-
```

Comment:
Removed this since it is a duplicate of the pytest run.

Reply:
@onuralpszr, was this the test that revealed the numpy import error a few weeks back? If so, did pytest not discover it?

Reply:
Let's remove old test and use poetry and your question answer is no it can't because i used pip to install

Reply:
What I am trying to say 2 things, we have new poetry/test actions which also check our "dependencies" and do "pytest" in case of "toml" file changes, we also have just a pytest by not using poetry at all. I said we can remove "old"  pytest/pip only one and use poetry + pytest (new action I recently I added). Second thing I notice in we are now using "poetry action" usually we do install via just pip but now we have full action, I was curious that why do we have that too. Because for me it is more complex and more dep

Reply:
In your second point, you're referring to this change, I believe.

```diff
- python -m pip install --upgrade pip
- pip install .
- pip install pytest
+ poetry install --with=test
```

If so, I am in favor of the new approach. When developing locally, we usually install with `poetry install --with dev` - that's what our docs advise. Even if using `poetry` introduces an extra layer, it's more consistent with our guidelines.